supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesBackendLifecycleActorFactory.scala

```diff
@@ -82,3 +58,29 @@ case class JesBackendLifecycleActorFactory(configurationDescriptor: BackendConfi
 
   override lazy val fileHashingFunction: Option[FileHashingFunction] = Option(FileHashingFunction(JesBackendFileHashing.getCrc32c))
 }
+
+object JesBackendLifecycleActorFactory {
```

Comment:
This seems to be a pattern in these changes of moving companion objects below the classes?